packages/engine/Source/Scene/CreditDisplay.js

```diff
@@ -270,10 +270,23 @@ function appendCss() {
     }
   );
 
-  const head = document.head;
+  function getShadowRoot(container) {
+    if (container.shadowRoot) {
+      return container.shadowRoot;
+    }
+    if (container.getRootNode) {
+      const root = container.getRootNode();
+      if (root instanceof ShadowRoot) {
+        return root;
+      }
+    }
+    return undefined;
+  }
+
+  const shadowRootOrDocumentHead = getShadowRoot(container) || document.head;
```

Comment:
We have a build-in function we prefer to use- `defaultValue`.
```suggestion
  const shadowRootOrDocumentHead = defaultValue(getShadowRoot(container), document.head);
```